### PR TITLE
Ld load libgobject 2.0.so.0

### PR DESCRIPTION
**Pull request by azubieta**
fix #99

### CODE REVIEW

File: src/libappimage/utils/DLHandle.h [hunk 1]
```diff
@@ -25,13 +25,37 @@ namespace appimage {
...
+            explicit DLHandle(std::initializer_list<const std::string> libNames, int mode) : handle(nullptr) {
+                for (const auto& item: libNames) {
+                    handle = dlopen(item.c_str(), mode);
+                    if (handle) {
+                        libName = item;
+                        break;
+                    }
+                }
+
+                if (!handle) {
```
Comment (TheAssassin) [inline src/libappimage/utils/DLHandle.h]: Please use explicit null(ptr) checks only.

File: src/libappimage/utils/IconHandle.cpp [hunk 2]
```diff
@@ -226,12 +226,13 @@ namespace appimage {
...
                  * @brief Load libgobject-2.0.so and resolve the symbol addresses required by the IconHandle.
                  *
+                 * Note: in Centos libgobject has 'libgobject-2.0.so.0' as soname.
                  * Mode comments:
                  * RTLD_LAZY - load the lib only the required symbols
                  * RTLD_NODELETE - do not unload the lib, as it wasn't designed to be used this way it
                  *                  will produce a big crash.
                  */
-                GLibOjbectHandle() : DLHandle("libgobject-2.0.so", RTLD_LAZY | RTLD_NODELETE) {
+                GLibOjbectHandle() : DLHandle({"libgobject-2.0.so", "libgobject-2.0.so.0"}, RTLD_LAZY | RTLD_NODELETE) {
```
Comment (TheAssassin) [inline src/libappimage/utils/IconHandle.cpp]: Which distros are covered by this list? This information helps when having the same issues again. Please document in-source.

Comment (azubieta) [reply src/libappimage/utils/IconHandle.cpp]: I can't tell all the distros that are covered. I left a _NOTE_ above saying that the Centos uses "libgobject-2.0.so.0".

File: src/libappimage/utils/DLHandle.h (hunk 1 above)
Comment (azubieta) [reply src/libappimage/utils/DLHandle.h]: Done

File: src/libappimage/utils/IconHandle.cpp (hunk 2 above)
Comment (azubieta) [reply src/libappimage/utils/IconHandle.cpp]: Comments where improved.

**Pull request merged**